package.json

```diff
@@ -6,7 +6,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "emulators:start": "cd ./firebase && firebase emulators:start --import=./seeds",
```

Comment:
Can we make this shorter? Something like `npm run ems`?

Reply:
I'd rather go for readability than convenience. ems doesn't mean anything to anyone especially if it's someone just starting out.